data/dc.json

```diff
@@ -1,32 +1,34 @@
 {
-    "contents": [
-        {
-            "title": "My first blog",
-            "body": "This is my first bog post",
-            "author": "John Doe",
-            "id": 1
-        },
-        {
-            "title": "My Second blog",
-            "body": "This is my second bog post",
-            "author": "John Doe",
-            "id": 2
-        },    {
-            "title": "My Third blog",
-            "body": "This is my third blog post",
-            "author": "John Doe",
-            "id": 3
-        },    {
-            "title": "My Fourth blog",
-            "body": "This is my fourth blog post",
-            "author": "John Doe",
-            "id": 4
-        },    {
-            "title": "My Fifth blog",
-            "body": "This is my fifth blog post",
-            "author": "John Doe",
-            "id": 5
-        }
-
-    ]
+  "contents": [
+    {
+      "title": "My first blog",
+      "body": "This is my first bog post",
+      "author": "Ayooluwa",
+      "id": "1"
+    },
+    {
+      "title": "My Second blog",
+      "body": "This is my second bog post",
+      "author": "Emmanuel",
+      "id": "2"
+    },
+    {
+      "title": "My Second blog",
+      "body": "This is my second bog post",
+      "author": "Tobi",
+      "id": "2"
+    },
+    {
+      "id": "86d6",
+      "title": "wewewew",
+      "author": "Ayooluwa",
+      "body": "rtrtrtrtrtrt"
+    },
+    {
+      "id": "4896",
```

Comment:
@Showcode24, weldone for this milestone submitted. While the functionality actually worked, you still need to fix a little glitch in your app before moving on to the next milestone.

Your json file is messed up and it's giving an error in the console after running your app because two objects with the same id (2)  are present in the file. Please remove one of the objects.


Secondly, An object with a non-numeric id value (86d6) is also found there. Kindly remove this as well.

After these removals, ensure your app is still working as expected and then you can resubmit this same pull request  for approval.